CMakeLists.txt

```
@@ -53,14 +54,19 @@ if (IS_DIRECTORY /lib/systemd/system)
   install(FILES emuccan.service DESTINATION /lib/systemd/system)
 endif()
 
+# install environment file
+if (IS_DIRECTORY /etc/default)
+  install(FILES emuccan.default DESTINATION /etc/default RENAME emuccan)
+endif()
+
 # Increment this for each rebuild (i.e. new kernel) using the same upstream ver
-set(CPACK_DEBIAN_PACKAGE_RELEASE "2")
+set(CPACK_DEBIAN_PACKAGE_RELEASE "1")
```

I think you're going to want to increment this, as the debian is sadly named: `emuccan-b202_-2.deb` and you'll want `emuccan-b202_-3.deb` so it upgrades.

```suggestion
set(CPACK_DEBIAN_PACKAGE_RELEASE "3")
```